react/hooks/useFetchMore.js

```diff
@@ -39,6 +40,11 @@ const handleFetchMore = (
       setLoading(false)
       fetchMoreLocked.current = false
 
+      if (!prevResult || !fetchMoreResult) {
+        updateQueryError.current = true
```

Comment:
a function ideally shouldn't modify an external object.

this works, but can be confusing. should at least be marked as a TODO, and fixed soon